internals/search.py

```diff
@@ -91,12 +93,28 @@ def process_recent_reviews_query() -> list[int] | Future:
   return future_feature_ids
 
 
+@dataclasses.dataclass
+class QueryContext:
+  now: datetime.datetime
+  current_stable_milestone: int
+
+  @classmethod
+  def current(cls) -> Self:
+    """Computes the "current" QueryContext based on ambient information."""
+    current_stable = None
+    for version in fetchchannels.get_omaha_data()[0]['versions']:
```

Comment:
This will slow down every query a bit, but it should be OK since it is usually only hitting redis.

Reply:
We could turn this into a function instead of a constant value, so it's only computed for queries that need it, but yeah, my theory was that redis is fast enough.